src/css/formulae.scss

```diff
@@ -145,7 +145,7 @@ $color-suffix: 00;
 }
 
 // border-radius
-@for $i from 0 through 16 {
+@for $i from 0 through 48 {
```

Comment:
Please revert this and create class of br-48

Reply:
Done